yt_dlp/extractor/vidio.py

```diff
@@ -56,9 +60,30 @@ def _real_extract(self, url):
             })
         video = data['videos'][0]
         title = video['title'].strip()
+        is_premium = video['is_premium']
+        if is_premium:
+            self.to_screen('Premier-exclusive video detected, switching to premier mode')
+            sources = self._download_json(
+                'https://www.vidio.com/interactions_stream.json?video_id=' + video_id + '&type=videos', display_id)
```

Comment:
```suggestion
            sources = self._download_json(
                'https://www.vidio.com/interactions_stream.json?video_id=%s&type=videos' % video_id,
                display_id, note='Downloading premium API JSON')
```

Feel free to change the actual `note` to whatever you want